tests/language/test_language_service.py

```diff
@@ -187,8 +187,9 @@ def test_language_flavor(self):
         Test that the service ignores files registered as being for non-PGSQL flavors
         """
         # If: I create a new language service
-        pgsql_params = LanguageFlavorChangeParams.from_data('file://pguri.sql', 'sql', 'pgsql')
-        mssqql_params = LanguageFlavorChangeParams.from_data('file://msuri.sql', 'sql', 'mssql')
+        pgsql_params = LanguageFlavorChangeParams.from_data('file://pguri.sql', 'sql', PG_PROVIDER_NAME)
```

Comment:
This test is for pg, lets add a test for MySQL language service  as well.

Reply:
Would you like me to move the MySQL flavor changes i do here to a separate test?